src/components/Tab.ts

```diff
@@ -0,0 +1,61 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { Clickable } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+
+const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    marginTop: '1rem',
+    height: '3.75rem',
+  },
+  button: (isActive: boolean) => ({
+    ...Style.tabBar.tab,
+    ...(isActive ? Style.tabBar.active : {}),
+    backgroundColor: undefined,
+    fontSize: 14,
+  }),
+};
+
+type TabProps = {
+  key: string;
+  title: string;
+  isValid: boolean;
+  activeTabKey: string;
+  onChangeTab: (arg0: string) => void;
```

Comment:
This argument could use a more descriptive name. As is, I don't know what to expect from this callback. Tab ID? The entire tab object?

```suggestion
  onChangeTab: (tabKey: string) => void;
```